server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -1365,39 +1368,6 @@ public static void updateClient(ClientRepresentation rep, ClientModel resource)
         resource.updateClient();
     }
 
-    public static void updateClientProtocolMappers(ClientRepresentation rep, ClientModel resource) {
```

Comment:
All the tests on Travis are broken, because this method was removed. Could you please elaborate more on why we have to remove this?